packages/react-vapor/src/components/blankSlate/tests/BlankSlateWithTable.spec.tsx

```diff
@@ -0,0 +1,19 @@
+import {shallowWithState} from 'enzyme-redux';
+import * as React from 'react';
+import {BlankSlateWithTable} from '../BlankSlatesHOC';
+import {IBlankSlateWithTableProps} from '../BlankSlateWithTable';
+
+describe('BlankSlateWithTable Tests', () => {
+    describe('<BlankSlateWithTable>', () => {
+        const defaultProps: IBlankSlateWithTableProps = {
+            title: 'test',
+        };
+
+        it('should not throw on mount and unmount', () => {
+            expect(() => {
+                const component = shallowWithState(<BlankSlateWithTable {...defaultProps} />, {});
+                component.unmount();
+            });
+        });
```

Comment:
There should be at least one unit test that verifies that a `tr` and `td` is rendered with the right `colSpan`